src/pykeen/triples/triples_factory.py

```diff
@@ -709,26 +719,49 @@ def from_path_binary(
         """
         path = normalize_path(path)
         logger.info(f"Loading from {path.as_uri()}")
-        data = torch.load(path)
-        return cls(**data)
+        return cls(**cls._from_path_binary(path=path))
+
+    @classmethod
+    def _from_path_binary(
+        cls,
+        path: pathlib.Path,
+    ) -> MutableMapping[str, Any]:
+        # load base
+        data = dict(torch.load(path.joinpath("state.pth")))
+        # load numeric triples
+        data["mapped_triples"] = torch.as_tensor(
+            pd.read_csv(path.joinpath("numeric_triples.tsv.gz"), sep="\t").values,
```

Comment:
maybe make the datatype in the CSV reading explicit?

Reply:
d86e301